common/arg_parser.h

```diff
@@ -141,13 +141,26 @@ class BitSet {
       bits_[i] = 0;
     }
   }
+  void fill(size_t bit = SIZE) {
+    for (size_t i = 0; i < NELEM(bits_); i++) {
+      bits_[i] = ~uint32_t(0);
+      if (i == (bit >> 5) && !!(bit & 31)) bits_[i] >>= 32 - (bit & 31);
+    }
+  }
   size_t popcount() const {
     size_t ret = 0;
     for (size_t i = 0; i < NELEM(bits_); i++) {
       ret += __builtin_popcount(bits_[i]);
     }
     return ret;
   }
+  size_t popcount_subset(size_t first, size_t last) const {
```

Comment:
I'm wondering if it would be easier (and faster) to do all this subset stuff with ands and ors.

Basically something like:

```cpp
 Bitset subset = ~(Bitset::All() << last) & (Bitset::All() << first);
 size_t popcount_subset = (subset & bitset).popcount();
```